styles.css

```diff
@@ -110,6 +116,8 @@ button:active {
 
 button:hover {
   cursor: pointer;
+  color: #ff6b00;
+  background-color: #fff;
 }
```

Comment:
@Gopxfs I can't even imagine repeating the whole week because of this single meticulous detail. I have one (extra) review left, so I can update the button now but what if another reviewer comes and says something else?

Reply:
> @Gopxfs I can't even imagine repeating the whole week because of this single meticulous detail. I have one (extra) review left, so I can update the button now but what if another reviewer comes and says something else?

Hey, @amexabee don't worry about this. Re-reviews focus only on what the previous reviewers said, so no surprises are going to appear. If they do say something different, it should have the [OPTIONAL] tag 😉.

Reply:
Based on this, your next reviewer should only look for your hover button effect. So, if you implement it correctly you are good to go 😄.